src/shared/components/layout/sidebar-modal/SidebarModal.scss

```diff
@@ -8,6 +8,7 @@
   grid-template-columns: 1fr 20px;
   grid-template-rows: 38px 1fr;
   height: 100%;
+  padding: 21px 0 0 12px;
```

Comment:
I see these are the same values as inside the `Sidebar` component. Does `SidebarModal` want to be inside the `Sidebar`, by any chance?